.changeset/twelve-tigers-stare.md

```diff
@@ -0,0 +1,274 @@
+---
+"generated-test-app": patch
+"rainbowkit-next-app": patch
+"with-next-siwe-iron-session": patch
+"with-next-rainbow-button": patch
+"with-next-siwe-next-auth": patch
+"with-next-custom-button": patch
+"with-next-wallet-button": patch
+"with-create-react-app": patch
+"with-next-app-i18n": patch
+"with-next-mint-nft": patch
+"@rainbow-me/create-rainbowkit": patch
+"@rainbow-me/rainbow-button": patch
+"with-next-app": patch
+"with-remix": patch
+"@rainbow-me/rainbowkit": patch
+"with-next": patch
+"with-vite": patch
+"example": patch
+"site": patch
```

Comment:
We should create separate changesets for rainbowkit, rainbow-button, create-rainbowkit, and the examples (that can be all in one)